.github/workflows/tests.yml

```diff
@@ -23,17 +24,17 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: 6
-          cache: 'npm'
```

Comment:
Would it be possible to put back this line, or is it somehow preventing the whole thing from working?

Reply:
It actually looking for lock file, so it would be possible after package-lock commit